dtfs-central-api/src/scheduler/jobs/create-utilisation-reports/index.ts

```diff
@@ -19,27 +19,28 @@ const isCurrentBankReportMissing = async (bank: Bank): Promise<boolean> => {
 };
 
 /**
- * Gets the banks that do not have reports
+ * Gets the banks which are visible in TFM utilisation
+ * reports and do not have any report in the database
  * @returns The banks which do not have reports
  */
-const getBanksWithoutReports = async () => {
-  const banks = await getAllBanks();
+const getBanksWithMissingReports = async (): Promise<Bank[]> => {
+  const banksVisibleInTfm = (await getAllBanks()).filter((bank) => bank.isVisibleInTfmUtilisationReports);
```

Comment:
Are we doing this filter elsewhere? Maybe worth adding a new repo method and doing the filter at the db level rather than in memory.

Reply:
We currently only do this specific logic in one other place and (as far as memory is concerned) there are a known and small number of banks each with a known size, so I'm happy leaving this as is for now with the caveat that this will probably be picked up as part of the SQL refactor